### DISCUSSION TITLE
关于打包vant问题

### DISCUSSION THREAD

**Discussion by wandbond** [youzan/vant#10740] [General]
各位大佬，泥萌好，我现在所在的团队正在研究学习vant的源码，我们使用vant 3.5.1 release版本的源码进行的研究，我们执行
```
pnpm build
```
报出这样的信息：
[error-msg.txt](https://github.com/youzan/vant/files/8966928/error-msg.txt)

查阅报错信息，很多 `TS2322: Type 'Element | ("" | Element | undefined)[]' is not assignable to type 'ReactNode'.
  Type 'Element' is not assignable to type 'ReactNode'.` 这样的错误，因此感觉是环境配置出了问题。

留意到有nodejs版本只支持^14.16.0，尝试使用`n`更换过nodejs版本到`14.19.3`，并删除pnpm-lock.yaml文件。但是问题依旧。

当前的环境信息为：
- node 14.19.3
- typescript: ~4.5.2

请问怎么配置才能正常打包呢？请问是否有相关的指引？

谢谢大佬。

**Comment by chenjiahan** [comment]
这个是之前项目里安装了 @types/react 导致的，可以拉最新的 dev 分支代码，没这个问题

**Reply by wandbond** [reply]
好的，我试试看，谢谢大佬~

**Reply by gnoixeijil** [reply]
万能的大佬
